src/locales/en.json

```diff
@@ -100,8 +100,8 @@
     "intimate_risk_warning": "We have applied a minimum Activity Risk for kissing due to the risk involved in exchanging fluids.",
     "intro": {
       "call_to_action": "Calculate approximate risk of contracting COVID",
-      "changes_warning": "<p>The microCOVID calculator now asks you to manually input an estimate of COVID levels, since we <0>no longer have access to automatic data</0> about COVID rates in different areas. Provide the best case estimates for your community in the calculator. Focus on case estimates and positivity as those provide the primary signal. Reach out to info@microcovid.org with any questions about the change!</p>",
-      "changes_warning_heading": "The microCOVID Project has switched to user-provided prevalence rates",
+      "changes_warning": "<p>microCOVID's data source is no longer providing updates on COVID cases. To use the calculator, you'll need to provide the COVID prevalence in your area. Here's how:</p><p>Go to the red 'Step 1' heading below. Look up <strong>weekly new cases per 100k</strong> in your area and enter it in the first box. Then enter '100,000' in the 'total population' box just below it. The other boxes should be left on their defaults.</p><p><em>In the US, you can get this info at covidactnow.org: Click your state on the map, then your county. The first number, labeled 'weekly new reported cases per 100k', is what you need.</em></p><p>We hope to have an easier interface in place soon. Thank you for bearing with us during this change. Contact info@microcovid.org with any questions.</p>",
```

Comment:
An observation: I don't think it's necessary to prescribe "per 100,000" and then to enter "100,000". It's also possible to enter total cases and total population. I do think it's useful to point out that it's possible to do it either way (and between the form and the guidance, it hints that either is possible).